File: quiz/src/main/java/com/example/quiz/service/question_option/QuestionOptionService.java
package com.example.quiz.service.question_option;

import com.example.quiz.base.baseInterface.BaseService;
import com.example.quiz.model.entity.question_option.QuestionOption;
import com.example.quiz.model.entity.question_option.QuestionOptionRequest;
import com.example.quiz.model.entity.question_option.QuestionOptionResponse;
import com.example.quiz.model.entity.question_option.QuestionOptionView;

public interface QuestionOptionService extends BaseService<QuestionOption, Long, QuestionOptionRequest, QuestionOptionResponse, QuestionOptionView> {
}

File: quiz-fe/src/services/question_option/question-option.service.ts
import { ENV } from "@/config/env";
import { BaseService } from "../BaseService";
import { QuestionOption, QuestionOptionRequest, QuestionOptionResponse, QuestionOptionView } from "./models";

export class QuestionOptionService extends BaseService<
QuestionOption,
QuestionOptionRequest,
QuestionOptionResponse,
QuestionOptionView
> {
     constructor() {
          super(ENV.API_URL, 'question-options');
     }
}

export const questionOptionService = new QuestionOptionService();